src/main/java/frc/robot/OI.java

```diff
@@ -55,14 +55,14 @@ private void setButtonAction(String name){
         private static final Joystick kJoystick = new Joystick(OI.kDriverJoystickPort);
         
           private static final Button kOrientationButton = Button.button1; //1, Toggle swerve orientation
-          private static final Button kZeroButton = Button.button3; //3, Zero the gyroscope
+          private static final Button kZeroButton = Button.Start; //3, Zero the gyroscope
+          private static final Button kOuttakeButton = Button.LB; //Right Bumper, run outtake
+          private static final Button kIntakeButton = Button.LT; //Right Trigger, run intake
           private static final Button kAlignForwardButton = Button.button4; //4, Align forwards
           private static final Button kAlignBackwardButton = Button.button2; //2, Align backwards
-          private static final Button kIntakeButton = Button.RT; //Right Trigger, run intake
-          private static final Button kOuttakeButton = Button.RB; //Right Bumper, run outtake
-          private static final Button kParkButton = Button.RB; //5, park the robot
-          private static final Button kToggleBalanceButton = Button.LB; //6, balance the robot
-
+          private static final Button kArmHighButton = Button.RB;
+          private static final Button kArmMidButton = Button.RT;
```

Comment:
arm mid - arm stow ( toggle ) - substation ( if gyro )

SHould triple check and see what this actually means.  We probably don't have a triple toggle